client/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Sunfish",
+  "version": "0.0.0",
+  "description": "A Siafile sharing App",
+  "repository": "https://github.com/NebulousLabs/Sunfish",
+  "devDependencies": {
+    "bower": "^1.3.1"
```

Comment:
Why bower when you can npm? It seems like eating cereal by duct-taping a second spoon to your first spoon.